client/irma.js

```diff
@@ -16,6 +16,28 @@ const UserAgent = {
     iOS: "iOS",
 };
 
+const ErrorCodes = {
```

Comment:
We should export this, so that users can do
`if (error === IRMA.ErrorCodes.ConnectionError.Initial) { ... }`
instead of 
`if (error === "CONNERR_INITIAL") { ... }`